engine/runtime-with-polyglot/src/test/scala/org/enso/interpreter/test/instrument/RuntimeVisualizationsTest.scala

```diff
@@ -315,7 +315,7 @@ class RuntimeVisualizationsTest
     val Some(Api.Response(_, Api.InitializedNotification())) = context.receive
   }
 
-  it should "emit visualization update when expression is computed" in {
+  it should "emit visualization update when expression is computed111" in {
```

Comment:
You can try it by running the visualization test

Reply:
How does this test simulate the situation? I don't see any change in the protocol? Any new message? What is the program that we run and what is the expression we evaluate at certain UUID?